lib/config.js

```diff
@@ -0,0 +1,28 @@
+const {app, BrowserWindow, dialog} = require('electron');
+const Configfile = require('electron-config');
+const configfile = new Configfile();
+const path = require('path');
+
+/**
+ * Handles the selection of the directories.
+ * @param {Object} current_window            - The window object of which to start the dialog from
+ * @param {Boolean} startup                  - To determinate wheather
+ * @return {folder}                          - Returns the folder
+ */
+function selectFolders(current_window, startup) {
+if (startup==false || !configfile.has('folder')) {
+var folder = dialog.showOpenDialog(current_window, {
+      title: 'Music folder',
+      defaultPath: app.getPath('music'),
+      properties: ['openDirectory']
+  });
+  configfile.set('folder', folder);
```

Comment:
Initially, the default music folder
```javascript
app.getPath('music')
```
should be included regardless. The user can then later remove this.